src/Workleap.Extensions.OpenApi.Analyzers/Internals/RoslynExtensions.cs

```diff
@@ -0,0 +1,44 @@
+namespace Workleap.Extensions.OpenApi.Analyzers.Internals;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Diagnostics;
+
+internal static class RoslynExtensions
+{
+    public static INamedTypeSymbol? GetBestTypeByMetadataName(this Compilation compilation, string typeMetadataName, string? assemblyName = null)
```

Comment:
Do you really need it? If so, I'm sure the implementation is not ok.

Reply:
Good point, i think we can just use GetTypeByMetadataName instead